engines/ospray/ispc/geometry/ExtendedSDFGeometries.ispc

```diff
@@ -161,28 +163,9 @@ struct ExtendedSDFGeometries
     int materialID;
 
     uint64 numExtendedSDFGeometries;
+    uniform bool useSafeIndex;
 };
 
-//////////////////////////////////////////////////////////////////////////////
-// These are functions needed to handle > 2GB buffers in ispc.
-
-#define __define_safe_index(T)                                              \
-    inline const uniform T* uniform safe_index(const T* uniform const base, \
-                                               const uniform uint64 index)  \
-    {                                                                       \
-        const uniform uint64 tSize = sizeof(uniform T);                     \
-        const uniform uint64 offsetBytes = tSize * index;                   \
-        return (const T* uniform const)((uint64)base + offsetBytes);        \
-    }                                                                       \
-                                                                            \
-    inline const uniform T* varying safe_index(const T* varying const base, \
-                                               const varying uint64 index)  \
-    {                                                                       \
-        const uniform uint64 tSize = sizeof(uniform T);                     \
-        const varying uint64 offsetBytes = tSize * index;                   \
-        return (const T* varying const)((uint64)base + offsetBytes);        \
-    }
-
 __define_safe_index(ExtendedSDFGeometry);
 __define_safe_index(uint64);
 __define_safe_index(int64);
```

Comment:
You should move the defines for uint64 and int64 somewhere else as they are generic types.

Reply:
I actually only need the uint64 type so I will remove the others and keep them here.